sbc_translation/models/sponsorship_correspondence.py

```diff
@@ -0,0 +1,41 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Stephane Eicher <eicher31@hotmail.com>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+import pdb
+
+from openerp import models, api
+
+
+class SponsorshipCorrespondence(models.Model):
+    """ This class intecept a letter befor it can be sending and sending
+    to local translate plateforme"""
+
+    _inherit = 'sponsorship.correspondence'
+
+    ##########################################################################
+    #                              ORM METHODS                               #
+    ##########################################################################
+    @api.model
+    def create(self, vals):
+        """ Create a message for sending the CommKit. """
+        pdb.set_trace()
+        sponsorship = self.env['recurring.contract'].browse(
+            vals['sponsorship_id'])
+        if sponsorship.reading_language.id == vals['original_language_id']:
+            letter = super(SponsorshipCorrespondence, self.with_context(
+                no_comm_kit=False)).create(vals)
```

Comment:
You don't need to set the context for putting the variable `no_comm_kit` to false. Not defining it will make the same result with less code.